c-sharp-server/c-sharp-client/Program.cs

```diff
@@ -6,48 +6,67 @@ class AuthDelegate : IDittoAuthenticationDelegate
 {
     public async void AuthenticationRequired(DittoAuthenticator authenticator)
     {
-        System.Console.WriteLine($"Login request");
+        Console.ForegroundColor = ConsoleColor.Red;
+        Console.WriteLine($"\nAuth required!! → try \"--login jellybeans\"");
+        Console.ResetColor();
     }
 
     public async void AuthenticationExpiringSoon(DittoAuthenticator authenticator, long secondsRemaining)
     {
-        System.Console.WriteLine($"Auth token expiring in {secondsRemaining} seconds");
+        Console.WriteLine($"Auth token expiring in {secondsRemaining} seconds");
     }
 }
 
-namespace Program {
+namespace Program
+{
 
     class App {
         static Ditto ditto;
-        static DittoSubscription subscription;
-        static DittoLiveQuery liveQuery;
+
+        static IDisposable dittoAuthObserver;
         static List<Task> tasks;
         static Boolean isAskedToExit = false;
 
-        public static void Main(string[] args) {
+        public static async System.Threading.Tasks.Task Main(params string[] args) {
             var appId = "f87f6d8c-1b51-46e2-83d6-d97825ebab71";
-            DittoLogger.SetMinimumLogLevel(DittoLogLevel.Debug);
             var identity = DittoIdentity.OnlineWithAuthentication(
                 appId,
                 new AuthDelegate(),
                 false,
                 "http://127.0.0.1:45002");
 
             ditto = new Ditto(identity);
+            ditto.DisableSyncWithV3();
+            ditto.Auth.Logout();
             DittoTransportConfig transportConfig = new DittoTransportConfig();
             transportConfig.Connect.WebsocketUrls.Add("ws://127.0.0.1:45002");
             ditto.TransportConfig = transportConfig;
             ditto.StartSync();
 
-            subscription = ditto.Store["tasks"].FindAll().Subscribe();
-            Console.WriteLine("Welcome to Ditto's Task App");
+            dittoAuthObserver = ditto.Auth.ObserveStatus((status) => {
```

Comment:
Since this is an `IDisposable` and if we don't need this when the methods ends, It would be recommended to have a `using` statement: 

```suggestion
            using var dittoAuthObserver = ditto.Auth.ObserveStatus((status) => {
```

With this, the `Dispose()` method will be called automatically when the variable gets out of scope.

Reply:
Thanks! [816be92](https://github.com/getditto/samples/pull/146/commits/816be92add9affc0d844c859a1fb60c2b36d5db1)